testing/test-helper.js

```diff
@@ -155,14 +152,11 @@ export class RequestBank {
   /**
    * Returns the URL for depositing a request.
    *
-   * @param id an unique identifier of the request.
+   * @param {number} id an unique identifier of the request in a test case.
    * @returns {string}
    */
   static getUrl(id) {
-    if (id.indexOf('/') >= 0) {
-      throw new Error('ID "' + id + '" should not contain "/"');
-    }
-    return `${REQUEST_URL}/deposit/${userAgent}-${id}/`;
+    return `${REQUEST_URL}/deposit/${id}/`;
```

Comment:
I see that in most use case, we only test only one request. Do you think it's better to simply have `1` as the default value?

Reply:
that only saves you 1 char. I'd rather be explicit here.

Reply:
: ) In  that case, I would argue that id can be a string. The number here is really confusing to me. But I'm also fine with whatever you put there.

Reply:
I agree with @zhouyx, the api seems strange. I liked the named request from before

Reply:
the named IDs were to prevent ID conflicts between tests, especially between test files.

after this change, each test case is strictly sandboxed, so we don't need to worry about that anymore. 

and we don't really need to use those ID to locate the failing test, as the failure in console already comes with a good description. 

as you see, in most test cases, we will only need one single ID. in some rare cases, you will need 2 or 3. So far I feel it's enough to have 1,2,3 to reference them. 

I also don't encourage a single test case to go too complicated to need descriptive string naming of those IDs. If there is one, very likely it's better be split into smaller test cases.

Reply:
I completely agree with you. That's why I proposed we provide a default value at first. And only when there are multiple requests, we can start to give them id names (which I still believe should be flexible and meaningful name)

Reply:
OK, I'm convinced to have a default. And people are free to give names if they're willing to.

Reply:
updated. the default ID is `undefined` :-)

Reply:
: ) Thanks.
BTW I just learned that undefined can be a key to the object.

Reply:
it's casted to string `undefined`